src/snapshot.rs

```diff
@@ -23,6 +23,25 @@ pub struct Snap {
     pub ledger_entries: OrdMap<LedgerKey, LedgerEntry>,
 }
 
+pub fn get_default_ledger_info() -> LedgerInfo {
+    LedgerInfo {
+        protocol_version: 19,
+        sequence_number: 0,
+        timestamp: 0,
+        network_id: vec![0u8],
```

Comment:
Instead of putting the default value for the LedgerInfo in this function that has to be explicitly called, I'd impl the `Default` trait for `SerializableState`, and have its default implementation set these values. Then it will be easy for callers of the functions to do `SerializableState::default()` to get a default state ready to go.

Reply:
I'm not sure about this. `SerializableState` isn't used outside on snapshot. The ledger entries and the ledger info are used separately, but they're combined in `SerializableState` so we can write them all to a file.